backend/docker/docker-compose-cere-mainnet.yml

```diff
@@ -123,7 +123,11 @@ services:
       RANKING_START_DELAY_MS: '1800000' # 30 min
       ACTIVE_ACCOUNTS_START_DELAY_MS: '300000' # 5 min
       ACTIVE_ACCOUNTS_CHUNK_SIZE: '100'
-
+      POSTGRES_USER: 'polkastats'
+      POSTGRES_HOST: 'postgres'
+      POSTGRES_DATABASE: 'polkastats'
+      POSTGRES_PASSWORD: 'polkastats'
+      POSTGRES_PORT: '5432'
```

Comment:
Why did you decide to add these vars? How it worked before without them?

Reply:
docker-compose was broken after db migrations PR.

This variables are needed for DB migrations config.

Reply:
What exactly was broken?

Reply:
after docker-compose start, migrations failed because of unset configurations

Reply:
you will need database.json if you want yo run db-migrate commands in command line

Reply:
We can remain `database.json` as it takes db credentials from environmental variables as we do for the whole application. When we run migration manually the environmental variables will be used.